zerobuf/Zerobuf.h

```diff
@@ -19,22 +19,22 @@ namespace zerobuf
  * Zerobuf objects can serialize/deserialize directly from their member storage
  * and from and to JSON.
  */
-class Zerobuf
+class ZEROBUF_API Zerobuf
```

Comment:
Do you recall which symbol was missing? Exporting the full class means exporting data fields as well as having external linkage of exported classes (might not be the case now). I'd try to export only what's needed.